<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 236, 227);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 16, 84);</string>
   </property>
   <property name="text">
    <string>Account Form</string>
   </property>
  </widget>
  <widget class="QPushButton" name="accountFormUpdateBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 170, 0);   /* base */
    color: white;
    border: 1px solid rgb(230, 150, 0);
    border-radius: 10px;
}

QPushButton:hover {
    background-color: rgb(255, 185, 40);  
}

QPushButton:pressed {
    background-color: rgb(230, 150, 0);   
}
</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QFrame" name="accountFormFrame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>441</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#accountFormFrame {
        border: 2px solid #1A1054;
        border-radius: 12px;
    
    }</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>181</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="fnameField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mnameField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>Middle Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnameField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sexField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>Sex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="birthdayField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="roleField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>Role</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="datehiredfield_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 16, 84);</string>
    </property>
    <property name="text">
     <string>Staff Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_staffnameField">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
        border: 2px solid #1A1054;
        border-radius: 5px;
        padding: 3px;
	background-color: rgb(255, 255, 255);
    }</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 16, 84);</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_usernameField">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
        border: 2px solid #1A1054;
        border-radius: 5px;
background-color: rgb(255, 255, 255);
        padding: 3px;
    }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 16, 84);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_passwordField">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
        border: 2px solid #1A1054;
        border-radius: 5px;
        padding: 3px;
background-color: rgb(255, 255, 255);
    }</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
